web-client/src/environments/environment.ts

```diff
@@ -1,41 +1,25 @@
-// This file can be replaced during build by using the `fileReplacements` array.
-// `ng build --prod` replaces `environment.ts` with `environment.prod.ts`.
-// The list of file replacements can be found in `angular.json`.
-
 import { Environment } from 'src/environments/types';
 
 export const environment: Environment = {
   production: false,
 
-  // Enable persistence for easier development.
+  // Enable persistence for demo purposes.
   persistAkitaState: true,
-
-  nautilusWalletServer: 'http://localhost:4200/api/nautilus/',
-  nautilusAssetServices: 'http://localhost:4200/api/asset-services/',
-  // See `proxyConfig` in `angular.json`, and `proxy.conf.json`
-  // Docs: https://angular.io/guide/build#proxying-to-a-backend-server
+  nautilusWalletServer: 'https://wallet-demo-api.ntls.io/',
+  nautilusAssetServices: 'https://wallet-demo-services.ntls.io/',
   algod: {
-    // XXX: Algodv2's parameter handling is a bit weird: the HTTP port must be passed separately.
-    baseServer: 'http://localhost/api/algorand',
-    port: 4200,
+    baseServer: 'https://testnet-algorand.api.purestake.io/ps2',
+    port: '',
     // FIXME: Development key
-    token: 'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa',
+    token: { 'X-API-Key': 'J7eo2jPb5m4OiBneIV6r0ajgRLeSaHqk3QplGETk' },
```

Comment:
aargh 

this wasn't supposed to get here.
Since I don't run the backend locally, I have to resolve to this, so Yes, this was a mistake from my side.